app/src/main/res/layout/activity_main.xml

```diff
@@ -20,24 +21,25 @@
         android:layout_marginTop="200dp"
         android:layout_marginBottom="20dp"/>
 
-    <com.google.android.material.textfield.TextInputLayout
-        android:id="@+id/quantity_parking_slots"
+    <EditText
+        android:id="@+id/quantity_parking_lots"
```

Comment:
Please review that all your views IDs comply with [Mobile Resources Style for Android](https://docs.google.com/document/d/1eImvh7-1tTKn6e2ivGDChqoD76nw4VzNk_ompvz4kJg/edit#heading=h.5r09p6gu0g3o)

Reply:
updated IDs with the following names:
`@+id/text_parking_size_instruction`
`@+id/text_parking_size_quantity_lots`
`@+id/button_parking_size_submit_parking_lots`